test/actualbetter.txt

```diff
@@ -0,0 +1,237 @@
+|| ===================================================
+|| ===================================================
+|| AddessBook Level 1 - Version 1.0
+|| Welcome to your Address Book!
```

Comment:
This file should not be version controlled. Use the git ignore feature to exclude files from version control.